src/js/events/MarathonActions.js

```diff
@@ -492,9 +492,13 @@ var MarathonActions = {
   },
 
   killPodInstances(pod, instanceIDs, force) {
-    let podID = encodeURIComponent(pod.getId());
+    let podID = pod.getId();
     let params = '';
 
+    if (podID[0] === '/') {
+      podID = podID.substr(1);
```

Comment:
Ok, Regex might be cleaner, but the performance optimiser inside me kicked-in 😄 

(I really had to crank the iterations up to 1,000,000 in order to see some numbers, so this optimisation really doesn't make any sense, but:)
```
substr: 21ms
regex: 45ms
```